pynamodb/models.py

```diff
@@ -675,6 +675,26 @@ def scan(cls,
             rate_limit=rate_limit,
         )
 
+    @classmethod
+    def record_exists(cls, hash_key, range_key=None):
```

Comment:
If we add this method, we should probably update models.pyi with its type information.

Reply:
Sounds good!